docs/scripts/devel-dependency-containers/docker-compose-all-sql.yml

```diff
@@ -25,7 +25,7 @@ services:
       POSTGRES_DB: opencast
   mariadb:
     container_name: opencast-mariadb
-    image: docker.io/library/mariadb:10.5.9
+    image: docker.io/library/mariadb:10
```

Comment:
Is it intentional that you selected `mariadb:10`? The most recent stable is `mariadb:11`?